src/shared/components/palette/base-field/field-read.component.spec.ts

```diff
@@ -152,4 +210,23 @@ describe('FieldReadComponent', () => {
     let fieldReadLabel = fieldReadLabelComponent.componentInstance;
     expect(fieldReadLabel.withLabel).toBe(true);
   });
+
```

Comment:
@johnbenjamin-hmcts - Lines 213 to 231... as above

Reply:
@Daniel-Lam - We had to add CaseEditComponent reference to FieldReadComponent, so that it can be accessed by the child component(s). These lines of code were to declare and add a reference to CaseEditComponent in the spec file.